### PR TITLE
St3 keymaps

### PR DESCRIPTION
**Pull request by EHLOVader**
Based on the extracted jars (sorry they seem to have duplicated here) 

This is the proposed change with each minor change separate. All of these commits can be squashed into one, but left them all separate for clarity on what changed for what.

### CODE REVIEW

File: osx-key-mappings/keymaps/Mac OS X 10_5_ Sublime.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    <keyboard-shortcut first-keystroke="shift control meta UP" />
+  </action>
+  <action id="ParameterInfo" />
+  <action id="Refactorings.QuickListPopupAction">
+    <keyboard-shortcut first-keystroke="control T" />
+    <keyboard-shortcut first-keystroke="meta alt ENTER" />
+  </action>
+  <action id="ReformatCode">
+    <keyboard-shortcut first-keystroke="meta alt L" />
+    <keyboard-shortcut first-keystroke="shift meta F" />
```
Comment (jcrben) [inline osx-key-mappings/keymaps/Mac OS X 10_5_ Sublime.xml]: Isn't this `Find in Files` in ST3 by default? Which would be `Find in Path` in PHPStorm if I'm not mistaken?

If this is going to be based on OSX ST3, I think it should stay faithful to the OSX ST3 defaults.

Comment (EHLOVader) [inline osx-key-mappings/keymaps/Mac OS X 10_5_ Sublime.xml]: It is, this was from a long time ago. It was never meant to be extensive. 

This does not contain the PR from [here](https://github.com/ShawnMcCool/PHPStorm-SpacePeacock/commit/4c0bb094367821cf98431285bc94a25cb9959cc5)

It is included in the other [PR](https://github.com/EHLOVader/PHPStorm-SpacePeacock/commit/135c7e317e01478026f32cc7e79e28d0e56fb3ed#diff-7a9f6f8b754d30a63dafbe2f530008cdR28) I recently updated, which this one depended on. I can update this once that is accepted. 

But it still appears the PR was a control shift f not command shift f like you suggest.

Comment (jcrben) [inline osx-key-mappings/keymaps/Mac OS X 10_5_ Sublime.xml]: I suggest changing it before it is merged. `Ctrl+Shift+F` could be mapped to refactor.

Comment (EHLOVader) [inline osx-key-mappings/keymaps/Mac OS X 10_5_ Sublime.xml]: Do your own PR, or comment on the source of that line. 

Unfortunately this project has been mostly binary file data, but that other PR of mine hopes to fix that, so we can properly do blame on lines and you can find the source of the problems.

**Comment by ShawnMcCool** [comment]
Thanks for this, if anyone wants to make any changes, please submit a PR and I'll merge them.
